source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -766,3 +777,34 @@ def complete_transfer(pt_journey, transfer_pool):
         if section.type != response_pb2.TRANSFER:
             continue
         transfer_pool.wait_and_complete(section)
+
+
+def prepend_first_and_append_last_coord(dp, origin, destination):
+    """
+    Insert/append origin and destination's coordinates to the geojson
+    """
+    if not dp or not dp.journeys or not dp.journeys[0].sections:
+        return
+
+    nb_coords = sum((len(sec.street_network.coordinates) for sec in dp.journeys[0].sections))
+    if nb_coords < 2:
+        return
```

Comment:
I don't really understand this check. Maybe add some comments ?

Reply:
it just checks the length of the `geojson` of the street network, there should be at least 2 points in a `geojson` logically